src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -113,7 +114,13 @@ public static LocalDate parseDate(String date) throws ParseException {
         requireNonNull(date);
         String trimmedDate = date.trim();
         DateTimeFormatter inputFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd");
-        return LocalDate.parse(trimmedDate, inputFormat);
+        LocalDate parsedDate;
+        try {
+            parsedDate = LocalDate.parse(trimmedDate, inputFormat);
+        } catch (DateTimeParseException e) {
+            throw new ParseException("Wrong input date format");
+        }
```

Comment:
I think this message can be replaced with the MESSAGE_INVALID_DATE_FORMAT constant from Messages.java.

Reply:
Done.